File: src/utils/googleMapsEmbedUrl.js
// Genera un link para incrustar Google Maps a partir de una direccion, sin necesitar API key.
module.exports = function googleMapsEmbedUrl(direccion) {
    if (!direccion) return null;
    return `https://www.google.com/maps?q=${encodeURIComponent(direccion)}&output=embed`;
};

File: utils/google_maps_embed_url.py
from urllib.parse import quote


def google_maps_embed_url(direccion):
    """Genera un link para incrustar Google Maps a partir de una direccion, sin necesitar API key."""
    if not direccion:
        return None
    return f'https://www.google.com/maps?q={quote(direccion)}&output=embed'
